DEVELOPMENT.md

```diff
@@ -80,3 +80,24 @@ Starting application at 127.0.0.1:3000
 
 And you'll be up and running. As you edit and save code, the `buffalo dev` command will notice and automatically
 re-compile and restart the server.
+
+# Run unit tests
+
+In order to run unit tests, services they depend on must be running first:
+
+```console
+make alldeps
+```
+
+and database created:
+
+```console
+buffalo db create
+buffalo db migrate up
```

Comment:
@adyach did you find that you needed these commands in order to run tests? I'm asking because the buffalo CLI should do this for you before it starts your tests.

Reply:
actually, it was suggested by @michalpristas, after I ran commands, the all the tests passed

Reply:
Ok, that's cool with me for now 👍 

Can you open up an issue for us to test whether these are actually needed?

Reply:
@arschles as far as I could formulate it https://github.com/gomods/athens/issues/452

Reply:
That's all good @adyach! Thanks for linking back to this thread as well! If anyone has questions, we can always refer back to here 🎉